### DISCUSSION TITLE
Best practice to name wallets

### DISCUSSION THREAD

**Discussion by haavardf** [eprbell/rp2#127] [Q&A] [answered]
I have named every single address that I am using with a specific name. Basically a letter and a running number if it belongs to the same private key. This doesn't look to work that great with RP2. Should I just look at my wallets as 'My Exchange' to simplify things. How have others solved this? Thank you.

**Comment by eprbell** [comment]
I use descriptive names containing attributes of the given wallet to remind me what that wallet is for. You can also use a color scheme (blue wallet, red wallet, etc.). I'm sure others have more ideas.

**Reply by haavardf** [reply]
Currently I just grouped all my wallets (with multiple addresses) using the same name, as in Wallet-Haavard. This is basically all offline wallets on the Bitcoin, Ethereum and Ripple blockchains. Does this has any implications on calculations, or is it only for my own references?

Happy to report I got my first report. RP2 is pretty strict, so I have some values going into negative values, so obviously something is not correct in my input data. Glad RP2 is picking up on it.

**Comment by eprbell** [comment, marked as answer]
I think it would be best to use different names for different wallets (even just a numeric suffix, if you can't come up with a better naming strategy). That way, in case of an audit, you know exactly which wallet any given transaction is coming from.

**Reply by haavardf** [reply]
The audit is a good point. I will go for this. Every wallet, but not every address as I started off with. Thank you. I asume it has no implications on any calculations then.

**Comment by eprbell** [comment]
Also, great to hear you got your first report! :-)

**Reply by haavardf** [reply]
I know. Very happy with that. No I just need to go through my transactions and figure out what's wrong. (Since I am transfering more then I have)
